### ISSUE TITLE
Problem with a Docker on Darwin

### ISSUE THREAD

**Issue by FZambia** [coursera/courseraprogramming#74]
Hello!

Just tried to use `courseraprogramming` tool on MacOS and got an error:

```
ERROR:root:Could not correctly pull in docker environment. Try running: eval "$(docker-machine env default)"
```

Docker version `17.03.1-ce, build c6d412e`, running suggested command did not help.

Mac OS 10.11.6

I looked at code - it assumes boot2docker. In method `kwargs_from_env` all enviroment variables (`DOCKER_HOST`, `DOCKER_CERT_PATH`, `DOCKER_TLS_VERIFY`) do not exist.

**Comment by ngarg-coursera** [comment]
@FZambia Can you please run it in the docker enabled terminal? I think on Mac OS, there is a way to run it within the terminal by directly running 'Docker quickstart terminal' which should basically connect with VirtualBox and do a bunch of stuff to set the environment variables. Please let me know if you still have trouble making it work.

**Comment by FZambia** [comment]
@ngarg-coursera Currently there is no need to run separate terminal with Docker on MacOS, we fixed this issue just working in the same way on `darwin` as on `linux` or `linux2` (i.e. like [this](https://github.com/coursera/courseraprogramming/blob/master/courseraprogramming/utils.py#L118))

**Comment by nrhodes** [comment]
This looks as though this was fixed in commit 318defc949b1dac22a85da9e4db7fb3c6103e6f3.  This created version 0.19.0.  Unfortunately, that version isn't available via pip.

**Comment by sashi-coursera** [comment]
Yeah, unfortunately - we are not able to push the updates to pip, please use steps here : https://github.com/coursera/programming-assignments-demo/tree/master/custom-graders#march-2018-alert, to use the latest courseraprogramming tool .
